test/api/questionnaire/samples.go

```diff
@@ -11,30 +11,30 @@ var (
 		Name:         "Questionnaire1",
 		Description:  "Questionnaire minimal sample 1",
 		Required:     true,
-		Thresholds:   assessment.Thresholds{},
-		RiskMessages: assessment.RiskMessages{},
-		Sections: []assessment.Section{
+		Thresholds:   api.Thresholds{},
+		RiskMessages: api.RiskMessages{},
+		Sections: []api.Section{
 			{
-				Order: uint2ptr(1),
```

Comment:
We made the Order required in the API, so no reason for it to continue to be a pointer.